atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -50,26 +50,28 @@ def __init__(self, name, max_concurrent_builds, enabled=True):
         self.enabled = enabled
 
 
+class ReactorConfigKeys(object):
```

Comment:
Really it's for use by the ReactorConfig class, which is concerned with fetching values for specific keys in the configuration file.

Other modules are expected to use the ReactorConfig instance returned by get_config(), which can provide methods for making queries against the configuration file.